packages/gatsby-script/.babelrc

```
@@ -0,0 +1,3 @@
+{
+  "presets": [["babel-preset-gatsby-package", { "browser": true }], "@babel/preset-typescript"]
```

I think you need esm to be true too, to make sure it's going to output smaller bundle size
```suggestion
  "presets": [["babel-preset-gatsby-package", { "browser": true, "esm": true }], "@babel/preset-typescript"]
```